### PR TITLE
Fix GAM Ext Authentication Serverless

### PR DESCRIPTION
**Pull request by sjuarezgx**
Issue 99934

### CODE REVIEW

File: dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs [hunk 1]
```diff
@@ -164,6 +167,104 @@ public override void Abort()
...
+		public ICollection<string> Keys { get { return m_headers.Keys; } }
+		public ICollection<StringValues> Values { get { return m_headers.Values; } }
+
+		public int Count { get { return m_headers.Count; } }
+
+		public bool IsReadOnly { get { return m_headers.IsReadOnly; } }
+
+		public void Add(string key, StringValues value)
+		{
+			m_httpResponseData.Headers.Add(key, value.AsEnumerable());
```
Comment (ggallotti) [inline dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs:209]: If the Key already exists, what would happen? Overwrite? Crash or make it duplicate? 

Example:
- Response.AddHeader("MyHeader", "value")
- Response.AddHeader("MyHeader", "value2")

**Review by ggallotti — COMMENTED** [review]
Just a minor comment, everything looks good!

I would rename the Issue to "Response.AddHeader support to Azure Serverless", because seems more generic issue.

File: dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs (hunk 1 above)
Comment (sjuarezgx) [reply dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs:209]: Several problems were fixed, not only the problema about the header. One problem was related to the path of the binaries.
I'll check what happens with duplicate headers..

Comment (sjuarezgx) [reply dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs:209]: It doesn't crash. If you have:
Response.AddHeader("MyHeader", "value")
Response.AddHeader("MyHeader", "value2")
The result is (value, value2)

**Pull request merged**